include/faker-cxx/science.h

```diff
@@ -98,5 +98,5 @@ FAKER_CXX_EXPORT Unit tempUnit();
  * current.
  * @endcode
  */
-FAKER_CXX_EXPORT Unit currentUnit();
+FAKER_CXX_EXPORT Unit currentUnit(Locale locale = Locale::en_US);
```

Comment:
please add doxygen comments aboout new param

Reply:
Yes i will add it

Reply:
I have added the comments